app/auto_responder.go

```diff
@@ -52,7 +52,7 @@ func (a *App) SendAutoResponse(channel *model.Channel, receiver *model.User) (bo
 		UserId:    receiver.Id,
 	}
 
-	if _, err := a.CreatePost(autoResponderPost, channel, false); err != nil {
+	if _, err := a.CreatePost(autoResponderPost, channel, false, true); err != nil {
```

Comment:
Shouldn't this be `false`? If the post is coming from the auto responder, that doesn't mean that the user is online.

I think there might already be clientside handling to prevent that, but it's probably best to be consistent here with what we tell the client.

Reply:
Sure, I just went with the current behavior to avoid breaking anything, and set it to false only for the creation of post. But will change this. Thanks.